doc: known_issues: add NCSDK-18746 (#9866)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

  The time returned by :ref:`lib_date_time` library becomes incorrect after one week elapses.
  This is due to an issue with clock_gettime() API.

.. rst-class:: v2-2-0

NCSDK-18746: LwM2M carrier library will fail to complete non-secure bootstrap when using a custom URI and the default security tag.
  If the LwM2M carrier library is operating in Generic mode (not connecting to any of the predefined supported carriers),
  and the :kconfig:option:`LWM2M_CARRIER_CUSTOM_URI` is set to connect to a non-secure server:

>   The LwM2M carrier library will attempt to retreive a PSK from :kconfig:option:`LWM2M_CARRIER_SERVER_SEC_TAG` even though a PSK should not be needed for the non-secure bootstrap.

Reviewer: typo `retrieve`

>   The PSK in this sec_tag is (corectly) not used, but reading the sec_tag causes the bootstrap to fail if the :kconfig:option:`LWM2M_CARRIER_SERVER_SEC_TAG` is default (0).

Reviewer: typo `correctly`

> NCSDK-18746: LwM2M carrier library will fail to complete non-secure bootstrap when using a custom URI and the default security tag.
>   If the LwM2M carrier library is operating in Generic mode (not connecting to any of the predefined supported carriers),
>   and the :kconfig:option:`LWM2M_CARRIER_CUSTOM_URI` is set to connect to a non-secure server:
>   The LwM2M carrier library will attempt to retreive a PSK from :kconfig:option:`LWM2M_CARRIER_SERVER_SEC_TAG` even though a PSK should not be needed for the non-secure bootstrap.
>   The PSK in this sec_tag is (corectly) not used, but reading the sec_tag causes the bootstrap to fail if the :kconfig:option:`LWM2M_CARRIER_SERVER_SEC_TAG` is default (0).
> 
>   **Workaround:** Assign any non-zero value to :kconfig:option:`LWM2M_CARRIER_SERVER_SEC_TAG`.
> 

Reviewer 2: ```suggestion
NCSDK-18746: LwM2M carrier library fails to complete non-secure bootstrap when using a custom URI and the default security tag
  If the LwM2M carrier library is operating in generic mode (not connecting to any of the predefined supported carriers), and the :kconfig:option:`CONFIG_LWM2M_CARRIER_CUSTOM_URI` is set to connect to a non-secure server, the library attempts to retrieve a PSK from :kconfig:option:`CONFIG_LWM2M_CARRIER_SERVER_SEC_TAG` even though a PSK is not needed for the non-secure bootstrap.
  The PSK in this ``sec_tag`` is not used, but reading the ``sec_tag`` causes the bootstrap to fail if the :kconfig:option:`CONFIG_LWM2M_CARRIER_SERVER_SEC_TAG` is set to the default value (0).

  **Workaround:** Assign any non-zero value to :kconfig:option:`CONFIG_LWM2M_CARRIER_SERVER_SEC_TAG`.

```
